try:
    i = 0
    while i < 5:
        i += 1
        print(i)
except Exception as e:
    print(f"Error : {e}")
